src/coreclr/tools/dotnet-pgo/CommandLineOptions.cs

```diff
@@ -265,6 +266,8 @@ void HelpOption()
                     }
                 }
 
+                syntax.DefineOption(name: "inherit-timestamp", value: ref InheritTimestamp, help: "If specified, set the output's timestamp to the max timestamp of the input files");
```

Comment:
Do we need an option for it at all? I'd leave "inherit" as default behavior.

Reply:
I don't really like this behavior as the default, I personally wouldn't expect a CLI tool that creates files to muck with the timestamps.

Reply:
ok, fair point